api/post.go

```diff
@@ -383,12 +388,14 @@ func handleWebhookEvents(c *Context, post *model.Post, team *model.Team, channel
 
 	relevantHooks := []*model.OutgoingWebhook{}
 	for _, hook := range hooks {
```

Comment:
Whoops, sorry I forgot to run `make check-server-style`.
It should be good now.